README.md

```diff
@@ -0,0 +1,219 @@
+            recipe_ID,     // ID of the asset to add
+            0               // ID of the asset to replace with the new one. 0 == none
+        );console.log("Add recipes to Master's book");
+    for (let i = 0; i < INITIAL_recipeS; i++) {
+        const recipe_ID = actual_recipe_id;
+        actual_recipe_id++;    // Update to the next ID
+        await cookBookInstance.addAssetToToken(
+            MASTER_BOOK_ID, // ID of the token that will receive the asset
+            recipe_ID,     // ID of the asset to add
+            0               // ID of the asset to replace with the new one. 0 == none
+        );                  // direct addition since the adder is the owner
+    }
+    }
+```
+
+Also Alice crafted her cookbook, but she did it in Argentina.
+
+```typescript
+// Alice mint her cookbook
+    await cookBookInstance.connect(ALICE).mint(
+        ALICE.address,          // address that will receive the token
+        1,                      // amount of tokens to mint
+        { value: pricePerMint } // price per token
+    );
+```
+
+She was inspired by different tastes and she created 2 recipes adding them **directly** to the cookbook.
+
+```typescript
+const INITIAL_ALICE_recipeS = 2;
+    for (let i = 0; i < INITIAL_ALICE_recipeS; i++) {
+        const recipe_ID = actual_recipe_id;
+        actual_recipe_id++;    // Update to the next ID
+        await cookBookInstance
+            .connect(ALICE)
+            .addAssetEntry(     // add a new asset to the collection (not binded to any token)
+                `ipsf://metadata/master_recipe_${recipe_ID}`  // metadata URI of the asset
+            );
+        await cookBookInstance.
+            connect(ALICE).
+            addAssetToToken(
+                ALICE_BOOK_ID,  // ID of the token that will receive the asset
+                recipe_ID,     // ID of the asset to add
+                0               // ID of the asset to replace with the new one. 0 == none
+            );
+    }
+```
+
+## recipe improvement - Asset replacement
+
+During his travel Master created a recipe dedicated to his student and he decided to **send** it to Alice, to add it to her cookbook.
+
+```Typescript
+// Master creates a new recipe and Alice adds it to her cookbook
+    await cookBookInstance
+        .connect(MASTER)
+        .addAssetEntry(     // add a new asset to the collection (not binded to any token)
+            `ipsf://metadata/master_recipe_${actual_recipe_id}`   // metadata URI of the asset
+        );
+```
+
+Alice **accepted** suddently the recipe gifted from her master...
+
+```Typescript
+await cookBookInstance.connect(ALICE).
+        addAssetToToken(
+            ALICE_BOOK_ID,      // ID of the token that will receive the asset
+            actual_recipe_id,  // ID of the asset to add
+            0                   // ID of the asset to replace with the new one. 0 == none
+        );
+    actual_recipe_id++;    // Update to the next ID
```

Comment:
The journey says the master sends it to alice but here it's alice adding it here self. I think it'd be better if Master was the one adding it, and then alice accepts it

Reply:
Change done.